docs/content/config/advanced/auth-ldap.md

```diff
@@ -6,37 +6,199 @@ title: 'Advanced | LDAP Authentication'
+- ...for incoming email, the inboxes which receive the email are chosen by the `USER`, `ALIAS` and `GROUP` filters.
+    - The `USER` filter specifies personal mailboxes, for which only one should exist per address, for example `(mail=%s)` (also see [`virtual_mailbox_maps`](http://www.postfix.org/postconf.5.html#virtual_mailbox_maps))
+    - The `ALIAS` filter specifies aliases for mailboxes, using [`virtual_alias_maps`](http://www.postfix.org/postconf.5.html#virtual_alias_maps), for example `(mailAlias=%s)`
+    - The `GROUP` filter specifies the personal mailboxes in a group (for emails that multiple people shall receive), using [`virtual_alias_maps`](http://www.postfix.org/postconf.5.html#virtual_alias_maps), for example `(mailGroupMember=%s)`
+    - Technically, there is no difference between `ALIAS` and `GROUP`, but ideally you should use `ALIAS` for personal aliases for a singular person (like `ceo@example.org`) and `GROUP` for multiple people (like `hr@example.org`).
+- ...for outgoing email, the sender address is put through the `SENDERS` filter, and only if the authenticated user is one of the returned entries, the email can be sent.
+    - This only applies if `SPOOF_PROTECTION=1`.
+    - If the `SENDERS` filter is missing, the `USER`, `ALIAS` and `GROUP` filters will be used in in a disjunction (OR).
+    - To for example allow users from the `admin` group to spoof any sender email address, and to force everyone else to only use their personal mailbox address for outgoing email, you can use something like this: `(|(memberOf=cn=admin,*)(mail=%s))`
 
-!!! example "saslauthd"
+???+ example
 
-    - `SASLAUTHD_LDAP_FILTER`
+    A really simple `LDAP_QUERY_FILTER` configuration, using only the _user filter_ and allowing only `admin@*` to spoof any sender addresses.
 
-!!! example "dovecot"
+    ```yaml
+    - ENABLE_LDAP=1
+    - LDAP_SERVER_HOST=ldap.example.org
+    - LDAP_SEARCH_BASE=dc=example,dc=org"
+    - LDAP_BIND_DN=cn=admin,dc=example,dc=org
+    - LDAP_BIND_PW=mypassword
+    - SPOOF_PROTECTION=1
+    
+    - LDAP_QUERY_FILTER_DOMAIN=(mail=*@%s)
+    - LDAP_QUERY_FILTER_USER=(mail=%s)
+    - LDAP_QUERY_FILTER_ALIAS=(|) # doesn't match anything
+    - LDAP_QUERY_FILTER_GROUP=(|) # doesn't match anything
+    - LDAP_QUERY_FILTER_SENDERS=(|(mail=%s)(mail=admin@*))
+    ```
+
+### `DOVECOT_*_FILTER` & `DOVECOT_*_ATTRS`
+These variables specify the LDAP filters that dovecot uses to determine if a user can log in to their IMAP account, and which mailbox is responsible to receive email for a specific postfix user.
+
+This is split into the folowing two lookups, both using `%u` as the placeholder for the full login name ([see dovecot documentation for a full list of placeholders](https://doc.dovecot.org/configuration_manual/config_file/config_variables/)). Usually you only need to set `DOVECOT_USER_FILTER`, in which case it will be used for both filters.
```

Comment:
```suggestion
This is split into the following two lookups, both using `%u` as the placeholder for the full login name ([see dovecot documentation for a full list of placeholders](https://doc.dovecot.org/configuration_manual/config_file/config_variables/)). Usually you only need to set `DOVECOT_USER_FILTER`, in which case it will be used for both filters.
```